src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -92,22 +94,65 @@ class WorkflowManagerActor(backend: Backend) extends Actor with CromwellActor {
     restartIncompleteWorkflows()
   }
 
+  private def reply[A](id: WorkflowId, fa: Future[A],
+                       onSuccess: (WorkflowId, A) => WorkflowManagerSuccessResponse,
+                       onFailure: (WorkflowId, Throwable) => WorkflowManagerFailureResponse): Unit = {
+    // Don't close over sender.
```

Comment:
Maybe all 32 million should have it. I actually fielded a question yesterday about what this comment meant. :smile: